cli/src/utils/tokenStorage.ts

```diff
@@ -48,10 +48,20 @@ export const TokenStorage = {
       return false;
     }
 
-    const decoded = jwtDecode<JWTPayload>(accessToken);
-    const currentTime = Math.floor(Date.now() / 1000);
+    // API keys don't expire
+    if (accessToken.startsWith("sk-")) {
+      return true;
+    }
+
+    try {
+      const decoded = jwtDecode<JWTPayload>(accessToken);
+      const currentTime = Math.floor(Date.now() / 1000);
 
-    return decoded.exp > currentTime;
+      return decoded.exp > currentTime;
+    } catch (_) {
```

Comment:
Let's never swallow errors like this, especially with multiple statements in the try catch